### ISSUE TITLE
using withMermaid breaks dev server in vitepress alpha 35

### ISSUE THREAD

**Issue by dominikcz** [emersonbottero/vitepress-plugin-mermaid#28]
As title says, when you use withMermaid() in your config.js like this:
```
// .vitepress/config.js
import { withMermaid } from "vitepress-plugin-mermaid";

export default withMermaid({
  // your existing vitepress config...
});
```
and you run 
```
npm run dev
```
you end up with an empty page and after some time (and 1530 requests) error in console:

> Uncaught SyntaxError: ambiguous indirect export: default mermaid.core.mjs:7:7

with `npm run build` everything seems to work correctly

**Comment by emersonbottero** [comment]
the problem is the mermaid version not the vitepress version. 
try installing mermaid 9.1 or lower.

**Comment by dominikcz** [comment]
OK, you are right and I can confirm that solution suggested in #24 (below) works... although I'd rather not have to use it

```
// docs/vite.config.ts or docs/vite.config.js
import { defineConfig } from "vite";

export default defineConfig({
  resolve: {
    alias: {
        "mermaid": "mermaid/dist/mermaid.esm.mjs",
    },
  },
});
```

**Comment by emersonbottero** [comment]
fixed in #32
